### DISCUSSION TITLE
Finished MortgageCalculator

### DISCUSSION THREAD

**Discussion by Felix-E-S** [Felix-E-S/T24S2T1EXTG08#5] [General]
I have finished the Mortgage Calculator. I have linked the exit button back to the main menu. Please let me know if you are happy with the design or if you would like me to change anything.

![image](https://github.com/user-attachments/assets/00f02052-45c9-4063-b25f-7a867bb5d9e8)
![image](https://github.com/user-attachments/assets/4143027e-fbdd-4eee-84ef-e7fd4ecb3e8f)

**Comment by StutzTAFE** [comment]
It looks good. Nothing that I can think of that needs to be changed.
